pkg/manager/service.go

```diff
@@ -279,6 +279,10 @@ func (m *Manager) Sync(ctx context.Context, req *rpc.SyncRequest) (resp *rpc.Bac
 	portReleaseChannel := make(chan interface{})
 	go func() {
 		<-portReleaseChannel
+		if !util.DetectHTTPServerAvailability(fmt.Sprintf("http://localhost:%d", port), 60, false) {
+			log.Errorf("Backing Image Manager: failed to wait for the HTTP server using port %v stopped, cannot release the port", port)
+			return
+		}
```

Comment:
Will it lead to the ports cannot be released correctly in case the HTTP server is somehow crashed?

Reply:
Yes. This may lead to some ports cannot get released.